src/integrationtests/integrationtestutils/TestEnvironment.ts

```diff
@@ -25,10 +25,40 @@ import { OrganizationsController } from "../../resources/organizations/controlle
 import { MongoDBOrganizationsRepository } from "../../resources/organizations/repositories/MongoDBOrganizationsRepository";
 import { OrganizationsRepository } from "../../resources/organizations/repositories/OrganizationsRepository";
 import { OrganizationsService } from "../../resources/organizations/services/OrganizationsService";
+import { UsersRepository } from "../../resources/users/repositories/UsersRepository";
+import { UsersService } from "../../resources/users/services/UsersService";
+import { MongoDBUsersRepository } from "../../resources/users/repositories/MongoDBUsersRepository";
 
 export class TestEnvironment {
-	ADMIN_TOKEN: string = "ADMIN_TOKEN";
-	USER_TOKEN: string = "USER_TOKEN";
+	createUser(
+		userIdentifier: string,
+		email: string,
+		permissionFlags: number,
+		organizationIdentifier: string,
+		role: string,
+	) {
+		return JSON.stringify({
+			identifier: userIdentifier,
+			email: email,
+			permissionFlags: permissionFlags,
+			organizationIdentifier: organizationIdentifier,
+			role: role,
+		});
+	}
+	ADMIN_TOKEN: string = JSON.stringify({
+		identifier: "adminID",
+		email: "admin@email.de",
+		permissionFlags: PermissionFlag.ADMIN_PERMISSION,
+		organizationIdentifier: "O_org_1",
+		role: "admin",
+	});
```

Comment:
Für diese Tokens könnten wir auch unseren `AuthUser`-Type benutzen, um typensicher zu arbeiten – z.B. so:

```suggestion
	private adminAuthUser: AuthUser = {
		identifier: "adminID",
		email: "admin@email.de",
		permissionFlags: PermissionFlag.ADMIN_PERMISSION,
		organizationIdentifier: "O_org_1",
		role: "admin",
	};
	ADMIN_TOKEN: string = JSON.stringify(this.adminAuthUser);
```

Reply:
Angepasst.